src/app/pages/country/country.component.ts

```diff
@@ -297,6 +298,59 @@ export class CountryComponent implements OnInit {
     'Curaçao': 'CW'
   };
 
+  public statesUSA = [
```

Comment:
This shouldn't be part of the component. I think it would be better if it's in another file labelled American states and then import it.

Reply:
I honestly put this in because i thought i would need it, however i'll probably just remove it since I parse it real time from the source data website.

Question for you, you have `countryCodes` in the component but you dont use it i dont think. Is that correct? Could it be removed as well?